<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLevelDialog</class>
 <widget class="QDialog" name="NewLevelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>行数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>列数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="horizontalGroupBox">
        <property name="title">
         <string>限制条件</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>步数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_3"/>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_44">
        <item>
         <widget class="QCheckBox" name="checkBox_40">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除单色蔬菜数量：西红柿</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_13"/>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>西红柿</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_15">
          <property name="text">
           <string>胡萝卜</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>卷心菜</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>玉米</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>茄子</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>指定分数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_9"/>
        </item>
       </layout>
      </item>
      <item row="23" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QCheckBox" name="checkBox_13">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除同色类型：西红柿</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QCheckBox" name="checkBox_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>生成特殊道具</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_43">
        <item>
         <widget class="QCheckBox" name="checkBox_39">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除单色蔬菜数量：胡萝卜</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_12"/>
        </item>
       </layout>
      </item>
      <item row="22" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QCheckBox" name="checkBox_12">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除同色类型：胡萝卜</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="15" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>消除所有地面</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="16" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>消除所有石块</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="17" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QCheckBox" name="checkBox_11">
          <property name="text">
           <string>消除所有特殊类型（只有特殊道具才可以消除）</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="14" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除所有冰块</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="checkBox_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除指定量坚果</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_10"/>
        </item>
       </layout>
      </item>
      <item row="10" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_45">
        <item>
         <widget class="QCheckBox" name="checkBox_41">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除单色蔬菜数量：卷心菜</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_14"/>
        </item>
       </layout>
      </item>
      <item row="11" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_46">
        <item>
         <widget class="QCheckBox" name="checkBox_42">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除单色蔬菜数量：玉米</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_15"/>
        </item>
       </layout>
      </item>
      <item row="12" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_47">
        <item>
         <widget class="QCheckBox" name="checkBox_43">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>消除单色蔬菜数量：茄子</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_16"/>
        </item>
       </layout>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="2">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="28" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="26" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_26">
        <item>
         <widget class="QCheckBox" name="checkBox_26">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除同色类型：茄子</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="24" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <widget class="QCheckBox" name="checkBox_14">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除同色类型：卷心菜</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="25" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QCheckBox" name="checkBox_25">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>消除同色类型：玉米</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="27" column="0">
       <widget class="QCheckBox" name="checkBox_16">
        <property name="text">
         <string>拯救所有哭泣的蔬菜</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QCheckBox" name="checkBox_17">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>消除可恶的瓢虫</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QCheckBox" name="checkBox_18">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>消除所有雪地</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_15"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="horizontalGroupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>一星:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_7"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>二星:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_6"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>三星:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_5"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NewLevelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NewLevelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
